src/code_reviewer/prompts.py

```diff
@@ -0,0 +1,17 @@
+"""Define default prompts."""
+
+with open("src/code_reviewer/system_prompt.md", "r") as file:
+    system_prompt_content = file.read()
+
+if system_prompt_content is None:
+  raise ValueError("File prompt is not found")
+
+SYSTEM_PROMPT = f"""
+{system_prompt_content}
+{{user_info}}
+System Time: {{time}}"""
+
+SYSTEM_PROMPT = SYSTEM_PROMPT.replace("{{user_info}}", "{user_info}")
+SYSTEM_PROMPT = SYSTEM_PROMPT.replace("{{time}}", "{time}")
```

Comment:
This is redundant, since `f` string will render `{{ foo }}}` as `{ foo }`.

Reply:
Thanks, this actually came from #7 at https://github.com/Moonsong-Labs/ai-nexus/pull/7/files#diff-4689798b5ce86b13516af7c0796eda5663e7f8c0e73901805c1a8100803f957a, so CC @GMKrieger.

That said, I think there's a bigger TODO which is to standardize across our agents how to format/include their prompts. Others have written it inside a giant string in Python code, which seems less than ideal to me.

Reply:
I think we can keep it chaotic for now, this allows to experiment different ways and find which one seems better before we refactor